docs/deploy.sh

```diff
@@ -125,22 +131,118 @@ redirects[eas-metadata/getting-started]=eas/metadata/getting-started/
 redirects[distribution/hosting-your-app/]=distribution/publishing-websites/
 
+# Redirects after adding Home to the docs
+redirects[next-steps/additional-resources/]=additional-resources/
+redirects[get-started/create-a-new-app/]=get-started/create-a-project
+redirects[guides/config-plugins/]=config-plugins/introduction/
+redirects[workflow/debugging/]=debugging/runtime-issues/
+redirects[guides/userinterface/]=ui-programming/user-interface-libraries/
+redirects[introduction/expo/]=core-concepts/
+redirects[introduction/faq/]=faq/
+redirects[workflow/expo-go/]=get-started/expo-go/
+redirects[errors-and-warnings/]=debugging/errors-and-warnings/
+redirects[develop/unit-testing/]=develop/unit-testing/
+redirects[guides/splash-screens/]=develop/user-interface/splash-screen/
+redirects[guides/app-icons/]=develop/user-interface/app-icons/
+redirects[guides/color-schemes/]=develop/user-interface/color-themes/
+redirects[development/introduction/]=develop/development-builds/introduction/
+redirects[development/create-development-builds/]=develop/development-builds/create-a-build/
+redirects[development/use-development-builds/]=develop/development-builds/use-development-builds/
+redirects[development/development-workflows/]=develop/development-builds/development-workflows
+redirects[debugging/]=debugging/runtime-issues/
+redirects[debugging/runtime-issue/]=debugging/runtime-issues/
+redirects[workflow/glossary-of-terms/]=more/glossary-of-terms/
+redirects[develop/development-builds/installation/]=develop/development-builds/create-a-build/
+rediects[get-started/errors/]=debugging/errors-and-warnings/
+rediects[develop/development-builds/parallel-installation]=build-reference/variants/
+redirects[home/develop/user-interface/safe-areas]=develop/user-interface/safe-areas/
+
+# Redirects after Guides organization
+redirects[guides/]=guides/overview/
+redirects[guides/routing-and-navigation/]=routing/introduction/
+redirects[guides/errors/]=debugging/runtime-issues/
+redirects[workflow/expo-cli/]=more/expo-cli/
+redirects[versions/latest/workflow/expo-cli/]=more/expo-cli/
+redirects[bare/hello-world/]=bare/overview/
+redirects[guides/using-graphql/]=guides/overview/
+redirects[guides/using-bugsnag/]=guides/overview/
+redirects[build/automating-submissions/]=build/automate-submissions/
+redirects[workflow/run-on-device/]=build/internal-distribution/
+redirects[archive/workflow/customizing/]=workflow/customizing/
+redirects[guides/building-standalone-apps/]=archive/classic-updates/building-standalone-apps/
+redirects[versions/latest/sdk/permissions#expopermissionscamera_roll]=guides/permissions/
+redirects[introduction/managed-vs-bare/]=archive/managed-vs-bare/
+redirects[push-notifications/using-fcm/]=push-notifications/push-notifications-setup/
+redirects[guides/using-custom-fonts/]=develop/user-interface/fonts/
+
+# Redirects reported from SEO tools list (MOZ, SEMRush, GSC, etc.)
+redirects[versions/v46.0.0/sdk/permissions#permissionresponse]=versions/v46.0.0
```

Comment:
```suggestion
redirects[versions/v46.0.0/sdk/permissions#permissionresponse]=guides/permissions/
```

Let's make this redirect future-proof, it would fail again after SDK 46 removal.